include/mqtt.hpp

```diff
@@ -0,0 +1,87 @@
+#ifndef INCLUDE_MQTT_HPP_
+#define INCLUDE_MQTT_HPP_
+
+#include <AsyncMqttClient.h>
+
+extern AsyncMqttClient mqttClient;
+
+void onMqttConnect(bool sessionPresent);
+void onMqttDisconnect(AsyncMqttClientDisconnectReason reason);
+void onMqttSubscribe(uint16_t packetId, uint8_t qos);
+void onMqttUnsubscribe(uint16_t packetId);
+void onMqttMessage(const char *topic, const char *payload,
+                   AsyncMqttClientMessageProperties properties, size_t len,
+                   size_t index, size_t total);
+void onMqttPublish(uint16_t packetId);
+
+template <class T>
+class Track {
```

Comment:
Can you please add a short description, what does the Track class is for and how does it work?